easybuild/easyconfigs/e/ENT/ENT-1.0.2.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Binary"
+
+name = 'ENT'
+version = '1.0.2'
+
+homepage = "http://dna.engr.uconn.edu/?page_id=25"
+description = """ENT is a highly scalable genotype 
+phasing algorithm based on entropy minimization. 
+ENT is capable of phasing both unrelated and related 
+genotypes coming from complex pedigrees."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://dna.engr.uconn.edu/~software/ent/']
+sources = ['ent_%(version)s.tar.gz']
+
+install_cmd = "tar xvzf ent_%(version)s.tar.gz &&"
+install_cmd += "cd %(builddir)s/ent_%(version)s &&"
+install_cmd += "make all &&"
+install_cmd += "cp -a %(builddir)s/ent_%(version)s/* %(installdir)s"
+
+dependencies = [
+     ('GCC', '4.7.2'),
```

Comment:
@rtripath89 why does this need `GCC` as a dependency, especially such an old version?

Wouldn't it be better to use GCC as a toolchain then rather than `dummy` to make this more explicit (e.g. in the module name)?